internal/gen.go

```diff
@@ -45,9 +45,9 @@ func fixQueryPlaceholders(engine, query string) (string, error) {
 	return newQuery, nil
 }
 
-func parseQueryReturn(tcf sql_types.TypeConversionFunc, col *plugin.Column) (*core.QueryReturn, error) {
+func parseQueryReturn(tcf sql_types.TypeConversionFunc, nullableHelpers *core.NullableHelpers, col *plugin.Column) (*core.QueryReturn, error) {
 	name := strcase.ToCamel(col.Name)
```

Comment:
I know this isn't part of the pr but looking at this now I think you could remove this line and just pass col.Name in the return val

Reply:
feel free to do it if you want otherwise I'll do it later

Reply:
Fixed!